plugins/kubernetes/app/models/kubernetes/role.rb

```diff
@@ -58,7 +58,7 @@ def self.seed!(project, git_ref)
         name = resource.fetch(:metadata).fetch(:labels).fetch(:role)
 
         # service
-        if service = config_file.service
+        if service = config_file.services.first
```

Comment:
I'm assuming it doesn't matter that we don't show all the services in the role UI.